src/themes/components/carousel.ts

```diff
@@ -96,6 +96,49 @@ const numbersPaginationButton = {
   },
 };
 
+const dotsPaginationButton = {
+  backgroundColor: 'transparent',
+  touchAction: 'manipulation',
```

Comment:
not sure this makes sense since you cannot pinch. Maybe `pan-x` would suit more?